### DISCUSSION TITLE
I don't understand why it's being rejected.

### DISCUSSION THREAD

**Discussion by cobaltlemo** [community/community#135678] [GitHub Education]
### Hi everyone,

I attempted to apply for the GitHub Education Pack based on a friend's recommendation, but my application has been denied, and this is the third rejection. I submitted all the required documents, including a certificate of enrollment issued by a South Korean government institution, my school's name, address, and my educational email, but I keep receiving the following message and my application is not being approved. How should I resolve this issue?

![캡처](https://github.com/user-attachments/assets/f4eb9946-0f5c-4d42-bf83-b8283c28d2be)

Translated into English using ChatGPT, and the translation may not be accurate.

**Comment by techy4shri** [comment]
Which email are you using for your account? Is it different from that of your academic email? And in your details, have you mentioned your irganisation as your institute? Those two could be the reason behind your rejections.

**Reply by cobaltlemo** [reply]
Thank you for your feedback. I'll apply it and try submitting my application again.

**Reply by techy4shri** [reply]
Let me know if you need further help! Or if the application is accepted :)

**Reply by cobaltlemo** [reply]
![캡처](https://github.com/user-attachments/assets/8dee76ca-d0aa-4149-b384-d2fb9508302d)

🥲

**Reply by cobaltlemo** [reply]
I've been rejected four times, and that message has appeared three times.

**Reply by techy4shri** [reply]
You have mentioned that you added your School ID, does it have any date verifying that you are still enrolled in the program?
I checked and it says this: 
Your ID should include a date that verifies your current enrollment. Make sure the image is clear and easy to read; if it looks blurry, please take a new photo and upload it again.

And what about the email you are using for the account? You also mentioned that the message has appeared 3 out of 4 times. What appeared the 4th time?

**Reply by cobaltlemo** [reply]
![캡처](https://github.com/user-attachments/assets/c3c26e00-83ae-43cb-8930-f3cc26167477)
When this text first appeared, it was during the initial rejection, and the photos I have uploaded since then are for the third and fourth attempts.

**Reply by cobaltlemo** [reply]
In the second attempt, the same message as the one shown in the photos from the third and fourth attempts also appeared

**Reply by techy4shri** [reply]
So does your ID has the date verifying that you are still part of the program?

**Reply by cobaltlemo** [reply]
Yes, I submitted a photo of the document saying that I am currently enrolled

**Reply by techy4shri** [reply]
Does it says something like: **valid till:  date** that is the date used to verify if your school ID is still valid.

**Reply by techy4shri** [reply]
hey there, did you get in?

**Comment by ansh3108** [comment]
You should try uploading your university's ID card and you should be good to go!

**Reply by cobaltlemo** [reply]
I attached and uploaded a certificate of enrollment issued by a government institution and a photo of my student ID showing the enrollment period. I hope it gets approved.
